src/controllers/account-controller.js

```diff
@@ -1,48 +1,213 @@
+        responseServer(response, error);
+    });
 }
 
 const postAccount = async (request, response) => {
     const {
         lastname,name,age,
-        dateBirth, email, URLPhoto,
-        username, password,role,
-        status,dateCreation, idCity
-    } = req.body;
+        dateBirth, email,
+        username, password,role, idCity
+    } = request.body;
+    let codeConfirmation = generateCode();
+    const idCityConverted  = mongoose.Types.ObjectId(idCity);
+    const dateNow = new Date();
+    const dateCreation = new Date(dateNow.getTime() - (dateNow.getTimezoneOffset() * 60000 )).toISOString().slice(0, 10);
     const account = new Accounts ({
         lastname: lastname,
         name: name,
         age: age,
         dateBirth: dateBirth,
         email: email,
-        URLPhoto: URLPhoto,
+        URLPhoto: "https:photo.com",
         username: username,
         password: password,
         role: role,
-        status: status,
+        status: 2,
+        codeConfirmation: codeConfirmation,
         dateCreation: dateCreation,
-        idCity: idCity
+        idCity: idCityConverted
     });
     account.password = await account.encrypPassword(account.password);
-    await account.save();
-    res.json({mensaje: 'El usuario se guardo correctamente'});
+    await account.save()
+    .then(async (account)  =>{  
+        const title = "Código de Confirmación de la cuenta"
+        const message = "Estimado usuario "+name+" "+lastname+
+        " para terminar el proceso de creación de su cuenta le enviamos su código de confirmación: "+ codeConfirmation;
+        await sendEmail(email,title,message);
+        response.status(StatusCodes.CREATED).json(account);
+    })
+    .catch(function (error){
+        responseServer(response, error);
+    });
 }
 
-module.exports = {getAccounts, postAccount}
\ No newline at end of file
+
+const putAccount = async (request, response) => {
+    const {
+        _id,lastname, name, age,
+        dateBirth, email, username, idCity
+    } = request.body;
+    const idAccount  = mongoose.Types.ObjectId(_id);
+    const idCityConverted  = mongoose.Types.ObjectId(idCity);
+    const queryAccount = {_id:idAccount};
+    const newValuesAccount = {lastname:lastname, name:name,
+    age:age, dateBirth:dateBirth, email: email, URLPhoto:"httpp.photo.com", idCity:idCityConverted, username:username};
+    Accounts.updateOne(queryAccount, newValuesAccount)
+    .then(function (document) {  
+        responseGeneral(response, StatusCodes.OK, "La cuenta se edito exitosamente");
+    })
+    .catch(function (error){
+        responseServer(response, error);
+    });
+}
+
+const patchAccount = async (request, response) => {
+    const {_id, status} = request.body;
+    const idAccount  = mongoose.Types.ObjectId(_id);
+    const queryAccount = {_id:idAccount};
+    const newValuesAccount = {status:status};
+    Accounts.updateOne(queryAccount, newValuesAccount)
+    .then(async (document) =>{  
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    .then(async (document) => {  
```